docs/toc.md

```diff
@@ -99,10 +100,13 @@ to know what's next -->
 ### [Getting started with .NET Core on Windows](core/tutorials/using-on-windows.md)
 ### [Getting started with .NET Core on macOS](core/tutorials/using-on-macos.md)
 ### [Getting started with .NET Core on Windows/Linux/macOS using the command line](core/tutorials/using-with-xplat-cli.md)
-### [Getting started with ASP.NET Core](core/tutorials/aspnet-core.md) 
+### [Writing Libraries with Cross Platform Tools](core/tutorials/libraries.md)
+### [Developing ASP.NET Core applications](core/tutorials/aspnet-core.md)
 ## [Deploying](core/deploying/index.md)
 ### [🔧 Deploying Applications](core/deploying/applications.md)
-### [Deploying Libraries](core/deploying/libraries.md)
+### [Creating a NuGet Package with Cross Platform Tools](core/deploying/creating-nuget-packages.md)
+### [How to Manage Package Dependency Versions for .NET Core 1.0](core/deploying/fastening-libraries.md)
```

Comment:
Thinking about it, this is a development topic, not a deployment topic. Right? You need this to get dotnet restore to work correctly. No?